### DISCUSSION TITLE
Fazer um menu de atributos

### DISCUSSION THREAD

**Discussion by kyuu123** [leogianfagna/IssueTracker#924] [Opinioes]
### Descrição do bug

apliquei um atributo de velocidade de ataque mas n mostra q ta com ele

### Reprodução

![image](https://github.com/user-attachments/assets/bf50d23b-da46-4188-8fbf-f4c619a67117)
![image](https://github.com/user-attachments/assets/ccebb94b-01c3-42d1-a368-003b6fd13e5e)
![image](https://github.com/user-attachments/assets/92d035ca-13c2-4785-b4f2-6c9b96c53e33)

### Servidor referido

Skyten

**Comment by kyuu123** [comment]
aproveitando. poderia ter um menuzinho mostrando qual o limite maximo de um atributo de tal categoria

**Comment by Micosedeunha** [comment]
Menu de atributos:

- abrir com /atributos
- mostrar os 7 atributos e seus máximos para cada nível
- mostrar quantos tem de cada atualmente com base no totem segurado na mão principal
- mostrar quantidade de xp para aplicar cada atributo
